docs/source/config_files.rst

```diff
@@ -163,3 +156,17 @@ used for installing libraries.
 
 To see an example R repository, visit our `R
 example in binder-examples <https://github.com/binder-examples/r/blob/master/runtime.txt>`_.
+
+
+``Dockerfile`` - Advanced environments
+======================================
+
+In the majority of cases, providing your own Dockerfile is not necessary as the base
+images provide core functionality, compact image sizes, and efficient builds. We recommend
+trying the other configuration files before deciding to use your own Dockerfile.
+
+With Dockerfiles, a regular Docker build will be performed.
+**If a Dockerfile is present, all other configuration files will be ignored.**
+
+See the `Binder Documentation <https://mybinder.readthedocs.io/en/latest/dockerfile.html>`_ for
```

Comment:
"See the Advanced Binder Documentation ..."

we are already in the Binder docs here after all :)